services/tenant-ui/frontend/src/components/layout/Sidebar.vue

```diff
@@ -57,27 +57,29 @@ const items = ref([
   //   to: { name: 'MyHeldCredentials' },
   // },
 
+  {
+    label: () => t('configuration.configuration'),
+    icon: 'pi pi-fw pi-file',
+    items: [
+      {
+        // label: () => t('configuration.schemasCreds.schemas'),
+        label: 'Schema Storage',
```

Comment:
No idea why typescript doesn't want THIS one to work in particular but all the others are fine... hard coding string for now.